src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -195,7 +195,9 @@ protected void handleUserException(Exception e) {
         if(printStackTraceOnUserExceptions()) {
             e.printStackTrace();
         } else {
-            System.err.println("Use -D" + STACK_TRACE_ON_USER_EXCEPTION_PROPERTY + "to print the stack trace.");
+            System.err.println("Use the system property "
+                            + STACK_TRACE_ON_USER_EXCEPTION_PROPERTY
+                            + " (--javaOptions '-DGATK_STACKTRACE_ON_USER_EXCEPTION=true') to print the stack trace.");
```

Comment:
It is important to mention `--javaOptions` explicitly, as that is a question we get asked a lot by our users.